tests/test_deployment/test_paths.py

```diff
@@ -0,0 +1,6 @@
+from flask_app.paths import get_combadge_path
```

Comment:
Regarding these tests - let's add relevant steps in CI (artifacts build, etc) so that they can run automatically.

Reply:
we can't. they use real infinidat hosts